### PR TITLE
chore(linux): Check in markdown help for Linux

### PR DESCRIPTION
**Pull request by darcywong00**
Since we're checking in the online help into the source repo
This cherrypicks the markdown help for Keyman for Linux in keymanapp/help.keyman.com#307

No build script updates needed since Keyman for Linux doesn't incorporate these files.

### CODE REVIEW

File: linux/help/reference/km-config.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+  <p>Launches Keyman Configuration for installing and showing information about Keyman keyboards</p>
+
+  <?php
+  syntax("km-config [-h] [--version] [-v] [-vv] [-i <INSTALL>]");
+  params("-h, --help", "optional", "show this help message and exit",
+    "--version", "optional", "show program's version number and exit",
+    "-v, --verbose ", "optional", "verbose logging",
+    "-vv, --veryverbose", "optional", "very verbose logging",
+    "-i keyboard.kmp, -install keyboard.kmp", "optional", "install .kmp package");
```
Comment (ermshiperete) [inline linux/help/reference/km-config.php:27]:
```suggestion
    "-i keyboard.kmp, --install keyboard.kmp", "optional", "install .kmp package");
```

File: linux/help/reference/km-package-get.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+$baselanguage = "language-bash";
+
+head([
+  'title' => "Command: km-package-get"
+]);
+
+?>
+
+<!-- TODO: Auto-generate from https://manpages.debian.org/unstable/keyman/km-package-get.1.en.html -->
```
Comment (ermshiperete) [inline linux/help/reference/km-package-get.php:13]: Do we have a task for these TODOs?

File: linux/help/reference/km-package-install.php [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    to download and install.
+  </p>
+  <h3>Tab Completion</h3>
+  <p>
+    You may use tab completion to find keyboard packages to install. The list comes from the network,
+    so there may be a small delay before tab completion will work the first time you use it. The list of packages
+    is cached for one week.
+  </p>
+  <p>
+    The list is the full list including keyboards that do not work on Linux, so they won't ncecessarily install.
```
Comment (ermshiperete) [inline linux/help/reference/km-package-install.php:43]:
```suggestion
    The list is the full list including keyboards that do not work on Linux, so they won't necessarily install.
```

**Review by ermshiperete — APPROVED** [review]

File: linux/help/reference/index.php [hunk 4]
```diff
@@ -0,0 +1,44 @@
+<?php
```
Comment (mcdurdin) [inline linux/help/reference/index.php:1]: We shouldn't be including .php files in the keyman repo

**Review by mcdurdin — CHANGES_REQUESTED** [review]

File: linux/help/reference/index.php (hunk 4 above)
Comment (darcywong00) [reply linux/help/reference/index.php]: @ermshiperete - I'll apply your suggestions to the php files in the help.keyman.com repo

File: linux/help/reference/km-package-get.php (hunk 2 above)
Comment (darcywong00) [reply linux/help/reference/km-package-get.php]: I don't think we have a task. That comment was from @glasseyes

File: linux/help/common/index.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+[https://packages.debian.org/keyman](https://packages.debian.org/keyman)
+
+
+## Q. How do I install a Keyman keyboard?
+**A.** Follow this [guide](../start/installing-keyboard) for installing Keyman keyboards.
+
+## Q. What's the relationship between Keyman for Linux and KMFL?
+**A.** Keyman for Linux and KMFL are released together in the Keyman
+[PPA](https://en.wikipedia.org/wiki/Ubuntu#Package_Archives), but are separate. The installation
+instructions above will install Keyman, not KMFL.
```
Comment (mcdurdin) [inline linux/help/common/index.md:24]:
```suggestion
[PPA](https://en.wikipedia.org/wiki/Ubuntu#Package_Archives), but are separate. The installation
instructions above will install Keyman, not KMFL. KMFL was developed as a black-box clone of Keyman
for Windows years ago; after Keyman was released as open source it became possible to share the 
code base between Windows and Linux. Keyman for Linux is the long-term supported solution.
```

**Review by mcdurdin — APPROVED** [review]

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 14.0.239-beta](https://keyman.com/downloads/releases/beta/14.0.239)

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.16-alpha](https://keyman.com/downloads/releases/alpha/15.0.16)

**Pull request merged**